Marlin/src/feature/power.cpp

```diff
@@ -120,7 +128,7 @@ void Power::check() {
 }
 
 void Power::power_on() {
-  lastPowerOn = millis();
+  lastPowerOn = millis() + 1; // add 1 because sometimes millis() return 0
```

Comment:
The `millis()` function should only return 0 before the timer starts, and then only once every 49 days.

Reply:
Not exactly...  I used mega2560 as COREXY machine with enable auto_power_control, and it just called powerManager.power_on() on or before the timer started, so it returns 0 all the time when boot the machine...

Reply:
Should I check for the 49 days???  I think it won't happen, only if the check was called on that exactly millisecond...

Reply:
With any possibility that `millis()` could be 0 (or 0xFFFFFFFF) I've modified the code to add 1 only if `millis()` returns 0.